doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -310,6 +310,10 @@ Bluetooth libraries and services
 
 * :ref:`bt_mesh` library:
 
+  * Added:
+
+    * The :kconfig:option:`BT_MESH_LIGHT_CTRL_AMB_LIGHT_LEVEL_TIMEOUT` Kconfig option that configures a timeout to reset the ambient light level to zero.
```

Comment:
timeout to reset
or timeout before resetting?

To me the second one sounds better. What do you think?
Strange I did not pick up on this yesterday.